### PR TITLE
Cloud FAQ and regions update

### PR DESCRIPTION
**Pull request by gemma-shay**
https://cockroachlabs.atlassian.net/browse/DOC-7621

@jramos-crl can you check that I got the regions right? the "known limitations" list you mentioned was removed recently, so I couldn't cross-check with it. also, I only added the gcp regions for Dedicated, but let me know if they go for Serverless too.

### CODE REVIEW

**Comment by jramos-crl** [comment]
@gemma-shay the regions are for Dedicated, you are right. But we are missing a few from the list that are available in the UI:

Virginia - us-east4
Hamina - europe-north1
Mumbai - asia-south1
Osaka - asia-northeast2

**Review by jramos-crl — APPROVED** [review]
LGTM

File: src/current/cockroachcloud/frequently-asked-questions.md [hunk 1]
```diff
@@ -81,7 +81,7 @@ The Cockroach Labs SRE team has direct access to {{ site.data.products.db }} clu
 
 ### How do I change the configurations on my cluster?
 
-Contact [Support](https://support.cockroachlabs.com/hc/en-us) to change your cluster configuration.
+You can [change your cluster's compute](cluster-management.html#change-compute-for-a-cluster), [add and remove nodes](cluster-management.html#add-or-remove-nodes-from-a-cluster), and [increase storage](cluster-management.html#increase-storage-for-a-cluster) from the {{ site.data.products.db }} Console. Due to cloud provider limitations, storage space cannot be removed from a node once added.
```
Comment (mdlinville) [inline src/current/cockroachcloud/frequently-asked-questions.md:84]: I think you can also make these changes using the API or Terraform, right?

**Review by mdlinville — APPROVED** [review]

File: src/current/cockroachcloud/frequently-asked-questions.md (hunk 1 above)
Comment (gemma-shay) [reply src/current/cockroachcloud/frequently-asked-questions.md:84]: double checking this with mark, but good point

**Pull request merged**